packages/blocks/styles/components/t-log-box.css

```diff
@@ -1,12 +1,22 @@
 .t-log-box {
   @apply relative
-         p-6;
+         p-6
+         bg-gray-800
+         text-gray-300
+         rounded-md
+         font-monaco
+         text-xs
+         leading-6;
```

Comment:
When we set style here, we can't apply any class to update this from our of component.
For example, if I NEED to change font or text size, then we have to change this style directly.

So I prefer to set this in components not css.